tests/integration/telemetry/stats/prometheus/customizemetrics/customize_metrics_test.go

```diff
@@ -48,26 +48,33 @@ func TestCustomizeMetrics(t *testing.T) {
 	framework.NewTest(t).
```

Comment:
Can you please make this test multi-cluster friendly? All the new tests should be multi-cluster friendly.. checkout other stats tests in this directory..